vm_phys.c

```diff
@@ -0,0 +1,251 @@
+#include <vm_phys.h>
+#include <queue.h>
+#include <kmem.h>
+#include <libkern.h>
+
+
+#define ORDER_SIZE(x) (1 << (x))
+
+/* this is a stub, should be moved to other file */
+#define panic(MSG...) do { kprintf(MSG); while(1); } while (0);
```

Comment:
Please do it now, as a separate and small PR. I'll import it into master and you'll rebase this branch.